include/os/linux/spl/sys/debug.h

```diff
@@ -54,8 +54,21 @@
 #define	__maybe_unused __attribute__((unused))
 #endif
 
+/*
+ * XXX: Without this, we see warnings from objtool during normal Linux builds
```

Comment:
```suggestion
 * Without this, we see warnings from objtool during normal Linux builds
```

No need for the XXX.